### PR TITLE
Fixing render tests on Windows by rolling back canvas package version

### PR DESCRIPTION
**Pull request by zhangyiatmicrosoft**
## Launch Checklist

node-canvas 2.11.0 has a bug that caused 3 render tests to fail on Windows.

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.
 - [x] Link to related issues.

### CODE REVIEW

**Comment by zhangyiatmicrosoft** [comment]
Appears to be a bug in the current version of node-canvas, which has been fixed but not released yet:
https://github.com/Automattic/node-canvas/commit/4c276e07f570ee6a727f88dcc67251823895c671

**Comment by zhangyiatmicrosoft** [comment]
All work fine now and running Windows in CI. Please merge

File: .github/workflows/test-all.yml [hunk 1]
```diff
@@ -43,16 +43,19 @@ jobs:
...
-    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           node-version: lts/*
       - run: npm ci
       - name: Start display server
-        if: runner.os == 'Linux'
+        if: matrix.os == 'ubuntu-latest'
```
Comment (rotu) [inline .github/workflows/test-all.yml:58]: Why change this? It should run even on other Linux images https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context

Comment (zhangyiatmicrosoft) [reply .github/workflows/test-all.yml]: Felt it is more consistent with the condition of <code>runs-on: ${{ matrix.os }}</code>
But I guess either way is fine. Changed back

**Review by HarelM — APPROVED** [review]

**Pull request merged**